subworkflows/local/germline_variant_calling.nf

```diff
@@ -94,6 +96,17 @@ workflow GERMLINE_VARIANT_CALLING {
         haplotypecaller_vcf = CONCAT_HAPLOTYPECALLER.out.vcf
     }
 
+    if ('deepvariant' in params.tools.toLowerCase()) {
+
+        DEEPVARIANT(
+            bam,
+            fasta,
+            fai)
+
+        haplotypecaller_vcf = DEEPVARIANT.out.vcf
+        haplotypecaller_gvcf = DEEPVARIANT.out.gvcf
```

Comment:
```suggestion
        deepvariant_vcf = DEEPVARIANT.out.vcf
        deepvariant_gvcf = DEEPVARIANT.out.gvcf
```

Reply:
doesn't make much sense to have output from deepvariant to go in haplotypecaller channel